README.md

```diff
@@ -9,7 +9,7 @@
 </p>
 
 Node.js is a JavaScript runtime built on Chrome's V8 JavaScript engine. For
-more information on using Node.js, see the [Node.js Website][].
+more information on using Node.js, see the [Node.js Website](https://nodejs.org).
```

Comment:
Could you instead add the link to bottom of this file? (e.g. undo just the `Node.js Website` change from https://github.com/nodejs/node/pull/27926).